File: concrete/src/main/java/pcb/uwu/unit/composite/electromagnetism/WebberUnit.java
package pcb.uwu.unit.composite.electromagnetism;

import pcb.uwu.units.composite.electromagnetism.MagneticFluxUnit;

import static pcb.uwu.unit.quantity.AmpereUnit.AMPERE;
import static pcb.uwu.unit.quantity.KiloGramUnit.KILOGRAM;
import static pcb.uwu.unit.quantity.MeterUnit.METER;
import static pcb.uwu.unit.quantity.SecondUnit.SECOND;

public class WebberUnit extends MagneticFluxUnit {

	public static final WebberUnit WEBBER = new WebberUnit();

	public WebberUnit() {
		super(KILOGRAM, METER, AMPERE, SECOND);
	}

	@Override
	public String getSymbol() {
		return "Wb";
	}

	@Override
	public String getSingularName() {
		return "webber";
	}

	@Override
	public String getPluralName() {
		return "webbers";
	}
}

File: concrete/src/main/java/pcb/uwu/unit/derived/electromagnetism/WebberUnit.kt
package pcb.uwu.unit.derived.electromagnetism;

import static pcb.uwu.unit.base.AmpereUnit.AMPERE;
import static pcb.uwu.unit.base.KiloGramUnit.KILOGRAM;
import static pcb.uwu.unit.base.MeterUnit.METER;
import static pcb.uwu.unit.base.SecondUnit.SECOND;

public class WebberUnit extends MagneticFluxUnit {

	public static final WebberUnit WEBBER = new WebberUnit();

	public WebberUnit() {
		super(KILOGRAM, METER, AMPERE, SECOND);
	}

	@Override
	public String getSymbol() {
		return "Wb";
	}

	@Override
	public String getSingularName() {
		return "webber";
	}

	@Override
	public String getPluralName() {
		return "webbers";
	}
}
